CONTRIBUTING.md

```
@@ -74,135 +71,160 @@ This command simulates the **Vercel environment locally**.
+```bash
+git checkout -b feature/short-description
+```
 
-# Issue Assignment (Mandatory)
+### Commit Message Convention
 
-- You must **comment "assign me"** before starting work
-- Wait for assignment from a maintainer
-- PRs without assignment may be **closed without review**
+**Format**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)